pyproject.toml.bak

```diff
@@ -9,7 +9,7 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.6"
```

Comment:
Nice... I should have looked at the entire commits first ;-). 

This looks good. Just squash the intermediate commits into the first one to reduce commit churns.